### DISCUSSION TITLE
Could the CRUD support multi-document transactions

### DISCUSSION THREAD

**Discussion by Epado** [mia-platform/community#224] [Q&A]
### Description

Here is the link to the mongo documentation 
https://www.mongodb.com/docs/manual/core/transactions/

**Comment by danibix95** [comment]
Hi @Epado, could you please elaborate more on the desired feature?

In particular:

- would it be possible to also provide a generic example on which this feature would bring great value when provided by CRUD Service?
- should this feature be available only over a single collection or multiple collections?
